vm/src/class.rs

```diff
@@ -112,6 +112,16 @@ pub trait PyClassImpl: PyClassDef {
                     .into();
             class.set_attr(identifier!(ctx, __new__), bound);
         }
+
+        if class
+            .slots
+            .hash
+            .load()
+            .map(|h| h as usize == unhashable_wrapper as usize)
+            .unwrap_or(false)
+        {
+            class.set_attr(ctx.names.__hash__, ctx.none.clone().into());
```

Comment:
are you going to delete `__hash__` or setting `None` value to it?

```rust
class.set_attr(ctx.names.__hash__, ctx.none.clone().into());  // this is A.__class__.__hash__ = None
class.set_attr(ctx.names.__hash__, None);  // this is del A.__class__.__hash__

```

Reply:
Setting Python-side `None` value.

If you delete `__hash__`, the class will be considered as Hashable (i.e. `isinstance(list(), Hashable) is True`) which is what we don't want.

Reply:
Thanks, I found even my comment to delete is wrong too 😂